### DISCUSSION TITLE
Is is possible to place camera preview inside a container instead of specifying xy?

### DISCUSSION THREAD

**Discussion by mannok** [capacitor-community/camera-preview#241] [Q&A]
Currently we can only specify xy of the camera preview image. May I know if it is possible to place the preview inside a predefined container, e.g. `<div>`?

**Comment by pbowyer** [comment]
Hi @mannok, thanks for your question. Due to the way Ionic/CapacitorJS works this is not possible.

Instead we have to use the `toBack: true` option to put the camera behind the page, and then change the Ionic app's CSS to expose the bit of the camera preview we want people to see.

**Reply by mannok** [reply]
Thanks @pbowyer

**Comment by ryaa** [comment]
If I'm not mistaken, when running on the web platform, you can set the html element id, which will be the container for the view. I'm wondering, if it is possible to force the web platform implementation on the native platform and therefore, set the parentId where the preview will be displayed.
